components/function-controller/pkg/apis/serverless/v1alpha1/function_types.go

```diff
@@ -24,6 +24,9 @@ type FunctionSpec struct {
 
 	// +kubebuilder:validation:Minimum:=0
 	MaxReplicas *int32 `json:"maxReplicas,omitempty"`
+
+	// +optional
+	PodLabels map[string]string `json:"podLabels,omitempty"`
```

Comment:
Hm... I'm not sure if `PodLabels` is a good name. Maybe just a `labels` as we have with `resources`, or create a new struct that will contain labels and resources as we don't want to use the `template` field (because we will need to block most of it...).

Reply:
I changed to `labels`, but we can thinking about this on our daily on Monday :)